app/src/definitions.ts

```diff
@@ -231,6 +231,11 @@ export interface AppPlugin {
    * @since 1.0.0
    */
   removeAllListeners(): Promise<void>;
+
+  /**
+   * Minimizes application
+   */
+  minimizeApp(): Promise<void>;
```

Comment:
can you move it up before all the listener related functions?
Also, can you include a line for `@since 1.1.0` as that would be the version where the feature will be included and mention that the feature is only available for Android?